src/llm/http_llm_calculator.cc

```diff
@@ -121,10 +125,10 @@ class OpenAIChatCompletionsRequest {
         if (seed.has_value())
             config.rng_seed = seed.value();
         if (frequencePenalty.has_value())
-            config.frequence_penalty = frequencePenalty.value();
+            config.frequency_penalty = frequencePenalty.value();
         if (presencePenalty.has_value())
             config.presence_penalty = presencePenalty.value();
-        config.do_sample = config.temperature > 0.0f && config.group_size == 1;
```

Comment:
Doesn't removing this part change functionality? 
@dkalinowski  

Seems like new `num_beams` is equivalent to old `group_size`

Reply:
It was confirmed with Ilya that:
- switching to **beam search sampling** was done by increasing the temperature and increasing group size at the same time
- switching to **multinomial sampling** was done by increasing the temperature and keeping the group size equal to 1

If nothing changed in CB lib regarding (@mzegla ?) sampling selection, yes, this change disables multinomial sampling entirely which is not expected @rasapala